Base/res/js/Spreadsheet/runtime.js

```diff
@@ -57,6 +57,56 @@ class Position {
         );
     }
 
+    range_up() {
+        if (this.row === 0) throw new Error(`No cells above this cell`);
+        const up_one = this.up(1);
+        let current_point = up_one;
+        for (
+            let point = current_point.up(1);
+            current_point.row !== 0 && typeof point.value() === "number";
+            point = current_point.up(1)
+        )
+            current_point = point;
+        return range(current_point.name, up_one.name);
```

Comment:
Shouldn't this contain the current cell as well?

Reply:
if I want to sum for example everything above this cell, why would I include the current cell?

Reply:
`range_up()` doesn't sound like `range_above()` to me, it sounds like `range_going_all_the_way_up()` which would...I think...naturally include this cell.

How about taking an `skip_above` param defaulted to 1?

Reply:
in what situation would you want to include the current cell in a formula?

Reply:
We can iterate on this in the future, for now I think it's fine